tgcc/raw/understanding-unets/train_unet_script.sh

```diff
@@ -3,21 +3,20 @@
 #MSUB -n 2                         # Number of tasks to use
 #MSUB -c 2                         # I want 2 cores per task since io might be costly
 #MSUB -x
-#MSUB -T 86400                      # Elapsed time limit in seconds
+#MSUB -T 21600                      # Elapsed time limit in seconds
 #MSUB -o unet_train_%I.o              # Standard output. %I is the job id
 #MSUB -e unet_train_%I.e              # Error output. %I is the job id
 #MSUB -q v100               # Queue
 #MSUB -Q normal
-#MSUB -m scratch,work
-#MSUB -@ zaccharie.ramzi@gmail.com:begin,end
-#MSUB -A gch0424                  # Project ID
-
+#MSUB -m work
```

Comment:
This script was used on a different HPC (TGCC instead of JZ): you need to rewrite it to fit to JZ

Reply:
done